src/caselawclient/models/identifiers/__init__.py

```diff
@@ -24,25 +33,46 @@ def __repr__(self) -> str:
     @classmethod
     @abstractmethod
     def validate_identifier(cls, value: str) -> bool:
+        """Check that any given identifier value is valid for this schema."""
         pass
 
 
 class Identifier(ABC):
-    """
-    A base class for subclasses representing a concrete identifier.
-    """
+    """A base class for subclasses representing a concrete identifier."""
 
     schema: type[IdentifierSchema]
+
+    uuid: str
```

Comment:
An identifier has a UUID -- I'm not sure how this is useful; do we want a prefix to mean we can disambiguate an arbitary Identifier UUID from a document UUID?

Reply:
Added an `id-` prefix, since this is basically zero cost.